linera-service/tests/end_to_end_tests.rs

```diff
@@ -612,6 +612,7 @@ async fn test_wasm_end_to_end_social_user_pub_sub(config: impl LineraNetConfig)
     Ok(())
 }
 
+#[ignore]
```

Comment:
Are all three tests failing in CI with all configurations? If it's e.g. only ScyllaDB, we should ignore only that test case instead (or maybe just comment it out).

Reply:
Correct, only the scylladb one has some issues.

Reply:
I indicated more clearly what tests are failing which should help for the debugging.